### DISCUSSION TITLE
I love you · poolp.org

### DISCUSSION THREAD

**Discussion by giscus** [poolpOrg/poolp.org#154] [Articles]
# I love you

Almost four years ago, your mom and I left home and rushed to the maternity hospital to give birth to your brother. As I closed the door to our home, I told your mom with a huge smile that the next time we’d open the door we’d no longer be only the two of us. This was the best day of our lives.
In December, we got the news that your mom was pregnant of you.

https://poolp.org/posts/2023-03-01/i-love-you/

**Comment by raionchani** [comment]
<3

**Comment by leisefuxX** [comment]
<3 long deep sincere cyberhug with tears in my eyes

**Reply by poolpOrg** [reply]
Thanks for your words, it's been months, lows and highs, time heals <3

**Reply by leisefuxX** [reply]
we can recover <3

**Reply by poolpOrg** [reply]
you went through a similar experience ?

**Reply by leisefuxX** [reply]
I went to the gates of hell and came back again. Cancer with stage 3b, heroin and cocaine addiction, loss of parents and and and ... I'm telling you from the bottom of my heart from way down deep: we can recover and we can heal again. I don't know what you've been through. i don't know what exactly has happened to you. but there is light and there is peace and one day you won't scream inside anymore and you won't cry anymore and you won't freak out anymore. one day it will be okay and you will accept it. 
i send you light and love, maybe from your neighboring country, maybe from the other corner of this planet. and i send you a big and long cyber hug. <3 

the silent foxX
silent foxX

**Reply by poolpOrg** [reply]
Yeah, time and patience heals all wounds.

Thanks for your kind words and sorry you went through hell even if life is better now :-(

Take care,
Gilles

**Reply by leisefuxX** [reply]
Thank YOU for your beautiful blog post <3
